Add CC (Número de Cartão de Cidadão, Portuguese Identity number)

davidmgvaz opened the pull request:
This adds CC (Número de Cartão de Cidadão, Portuguese Identity number)

DimitriPapadopoulos commented on stdnum/pt/cc.py:
```diff
@@ -0,0 +1,106 @@
+
+"""CC (Número de Cartão de Cidadão, Portuguese Identity number).
+
+The Portuguese Identity Number is alfa numeric with the following format:
```
alphanumeric

davidmgvaz replied:
Correct in last coomit

DimitriPapadopoulos commented on stdnum/pt/cc.py:
```diff
@@ -0,0 +1,106 @@
+
+
+def is_valid(number):
+    """Check if the number is a valid VAT number."""
```
... a valid cartao de cidadao number

davidmgvaz replied:
Corrected in last commit

davidmgvaz commented:
Hi @DimitriPapadopoulos,

I have already corrected the two typos you found and also split the calck_check_digit to an autonomous function, as I saw that was the standard practice

arthurdejong commented:
Hi @davidmgvaz 

Thanks for the contribution. Could you provide some test numbers that can be used to confirm the implementation? For most number formats we have a number of tests with valid numbers to be able to validate the correctness of the calculations.

arthurdejong commented:
Also a part of the number is a Número de Identificação Civil which has its own check digit. Do you have any idea (or pointers) how to validate that number?

magamig commented:
@arthurdejong any particular reason to not update the README?

arthurdejong commented:
The README and documentation are updated as part of the release process with [update/numlist.py](https://github.com/arthurdejong/python-stdnum/blob/master/update/numlist.py).
